ArraysAndLists/Program.cs

```diff
@@ -19,49 +20,106 @@ public static class ArraysAndLists
         // If more than two people like your post, it returns: "[Friend 1], [Friend 2] and [Number of Other People] others liked your post."
         public static string FacebookLikes(params string[] names)
         {
-            // Delete the below line when you are writing your code, don't forget to use your
-            // own return statement.
-            return default;
+           
+           
+            
+            string strMessage = string.Empty;
+          
+           if(names.Length == 0)
+			{
+                return strMessage = string.Empty;
+			}
+           if (names.Length ==1 )
+			{
+                return strMessage = String.Format("{0} liked your post.", names[0]);
+			}
+           if (names.Length ==2)
+			{
+                return strMessage = String.Format("{0} and {1} liked your post.", names[0], names[1]);
+			}
+           if (names.Length > 2)
+			{
+                return strMessage = String.Format("{0}, {1} and {2} others liked your post.", names[0], names[1], (names.Length - 2));
```

Comment:
All of the returns in these lines are redundant. You can do the exact same thing w/o them.